app/views/admin/admins/_form.html.haml

```diff
@@ -3,7 +3,7 @@
   = f.input :given_names, input_html: {maxlength: User::MAX_NAME_LENGTH}
   = f.input :family_name, input_html: {maxlength: User::MAX_NAME_LENGTH}
   = f.input :email
```

Comment:
for whatever reason, it's not required. It either tries to autocomplete both or neither